docs/howto.md

```diff
@@ -0,0 +1,23 @@
+# How to use the Web Scraper panel
+
+You can boost search result relevance by taking advantage of this feature. Read more information on the official Coveo documentation site - [Web Scraping Configuration](https://docs.coveo.com/en/1874/index-content/web-scraping-configuration).
+
+_To install this extension in your Chrome browser, follow these [installation steps](./install.md)._
+
+## How-to Guide
+
+1. Open `View > Developer > Developer Tools` or `Option + ⌘ + J` (on macOS), `Shift + CTRL + J`(on Windows/Linux).
+   ![Developer Tools](./devtools.png)
+1. Find a Web Scraping tab (like show on the screenshot above)
```

Comment:
Just a minor grammatical error.
The sentence in the bracket should be - **like shown on the screenshot above**

Reply:
Thanks. Fixed it directly in the `master` branch. Also changed 'above' to 'below'. 😄